assignment1.txt

```diff
@@ -0,0 +1,5 @@
+1. If the code has been executed in the way it was meant to, the function return 0; tells the OS so. If the return value is 1, it implies there was a problem in the execution of the program 
+2. x is a floating point number, which means it has a decimal value. The use of the command %d instructs the program to output an integer value, which x is not
+3.The equation would first divide m/r, which would result in the wrong answer
```

Comment:
Correct! You can fix this with parentheses. The line of code is also incorrect because of two more errors - try and find them.